broker-cli/broker-daemon-client/index.js

```diff
@@ -66,7 +66,7 @@ class BrokerDaemonClient {
 
     if (this.disableAuth) {
       // TODO: Eventually allow broker daemon client to use the cli's logger
-      console.warn('disableAuth is set to true. The CLI will try to connect to the daemon without ssl')
+      console.warn('`disableAuth` is set to true. The CLI will try to connect to the daemon without SSL/TLS'.yellow)
```

Comment:
it's kind of weird to use `colors` without requiring them in this file

Reply:
I like the use of color here (because grey text isn't as interesting of a warning). ill add colors to the top JIC